dogpile/cache/backends/memcached.py

```diff
@@ -220,16 +220,6 @@ def delete_multi(self, keys):
 class MemcacheArgs(GenericMemcachedBackend):
     """Mixin which provides support for the 'time' argument to set(),
     'min_compress_len' to other methods.
-
-    :param dead_retry: Number of seconds memcached server is considered dead
-       before it is tried again..
-
-     .. versionadded:: 1.1.7
```

Comment:
OK this can't be removed unless it is absolutely clear that this argument can never work on any backend, ever.  this has been released